docs/source/docs/providers/aws/instance-parameters.md

```diff
@@ -1,84 +1,87 @@
 # Instance Parameters
 
-Instance parameters are part of the [configuration file], but for each provider they are different. 
```

Comment:
Could you, please, revert back changes for the lines that are not related to this feature?